tests/ai_hub/model_registry/rest_api/conftest.py

```
@@ -486,25 +502,44 @@ def model_registry_connection_secret(
 
-
 @pytest.fixture(scope="class")
 def model_registry_serving_runtime(
     admin_client: DynamicClient,
     model_registry_deployment_ns: Namespace,
 ) -> Generator[ServingRuntime, Any, Any]:
-    """Create an OVMS ServingRuntime from the cluster template for registered models."""
+    """Create an vLLM CPU ServingRuntime from the cluster template for registered models."""
+
+    if get_cluster_architecture() == "s390x":
+        LOGGER.info(f"Switching to VLLM_CPU_Z.")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)